<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qmutemaster</class>
 <widget class="QFrame" name="qmutemaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mute Master</string>
  </property>
  <property name="toolTip">
   <string>The Mute Master allows for easier viewing of mute groups.</string>
  </property>
  <property name="whatsThis">
   <string>This is the Mute Master</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>6</y>
     <width>417</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="m_group_table">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>340</height>
       </size>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>32</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>16</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>18</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QSpinBox" name="m_pattern_offset_spinbox">
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>When the set-size is 64 and 96 instead of 32,
this control allows paging the Group Patterns
grid.  A future feature.</string>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Saves the current mute groups to a file in the session
configuration directory.</string>
        </property>
        <property name="text">
         <string>Save All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_load">
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_up">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>52</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move the selected set up in the list.</string>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_down">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>52</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move to the next mute group.</string>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>322</width>
     <height>275</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mute-Groups</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="setGridLayout" rowminimumheight="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Group Patterns</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="patternGridLayout" rowminimumheight="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mutes File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="m_mute_basename">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Provides the base name for the mute-group file.
If a path is present, it is stripped.  The file is always
saved to the session configuration directory.</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Write Format</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="m_radio_binary">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>This setting causes the mute statuses to be saved
as stanzas of 1's and 0's.  This is the default setting.</string>
          </property>
          <property name="text">
           <string>Binar&amp;y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="m_check_to_mutes">
          <property name="toolTip">
           <string>If checked, the mute-groups are saved to the
&quot;Mutes File&quot; (*.mutes) when the application
exits.</string>
          </property>
          <property name="text">
           <string>To Mutes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="m_radio_hex">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>If set, the mute statuses are saved in hexadecimal
format (e.g. 0xFF).  Useful if the set-size is larger
than the default of 8 x 4.</string>
          </property>
          <property name="text">
           <string>He&amp;x</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="m_check_to_midi">
          <property name="toolTip">
           <string>If checked, the mute-groups are saved in the MIDI
file when it is saved.</string>
          </property>
          <property name="text">
           <string>To MIDI </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="m_check_toggle_active">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>If checked, toggling off a mute-group toggles only
the patterns active in that group; patterns unmuted
by the user will stay on.  The legacy behavior is to
adhere strictly to the mute-group.</string>
        </property>
        <property name="text">
         <string>Toggle Only Active</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_check_load_mutes">
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>If checked, load the mute-group section in the
'mutes' file.</string>
        </property>
        <property name="text">
         <string>Load Mutes from File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="m_button_trigger">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>When active, the mute-group button in this
tab can be used to toggle the mute-groups.</string>
        </property>
        <property name="text">
         <string>Trigger Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_clear_all">
        <property name="toolTip">
         <string>Removes all mute-groups from the mute-group setup.</string>
        </property>
        <property name="text">
         <string>Clear All Mutes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_button_set_mutes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Saves the current mute-group modifications to
the mute-group settings.  One still needs to save
the MIDI file as applicable to preserve this mute-group.</string>
        </property>
        <property name="text">
         <string>Update Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
